index.bs

```diff
@@ -5735,15 +5611,12 @@ partial interface MLGraphBuilder {
   <div class=algorithm-steps>
     1. If |splits| is an {{unsigned long}}, and |input|'s [=MLOperand/shape=][|options|.{{MLSplitOptions/axis}}] % |splits| is not 0, then [=exception/throw=] a {{TypeError}}.
     1. If |splits| is a sequence of {{unsigned long}}, and the sum of its elements is not equal to |input|'s [=MLOperand/shape=][|options|.{{MLSplitOptions/axis}}], then [=exception/throw=] a {{TypeError}}.
-    1. If any of the following sub-steps fail, [=exception/throw=] an "{{OperationError}}" {{DOMException}}.
+    1. *Make graph connections:*
         1. Let |output| be the result of [=copying an MLOperand=] given |input|.
```

Comment:
Split should have multiple outputs in a list.

Reply:
Done - this turned into a substantial addition but since you called it out I did it in this PR. Please verify I got the algorithm right?

Reply:
https://github.com/webmachinelearning/webnn/commit/2246810db4b46ec70c457813ea17cca6d3e34fc0 looks good to me. Thanks!